data/secscan_model/secscan_v4_model.py

```diff
@@ -401,6 +404,45 @@ def should_skip_indexing(manifest_candidate):
                     dur_ms = get_epoch_timestamp_ms() - manifest.created_at
                     dur_sec = dur_ms / 1000
                     secscan_result_duration.observe(dur_sec)
+
+                    if features.NOTIFY_NEW_PUSH_VULNS:
+                        # if there are "new" vulnerabilities found, take the first one and add it to the notification queue
+                        try:
+                            vulnerability_report = self._secscan_api.vulnerability_report(
+                                manifest.digest
+                            )
+                        except APIRequestFailure:
+                            vulnerability_report = None
+
+                        try:
+                            found_vulnerabilities = vulnerability_report["vulnerabilities"]
+                        except TypeError:
+                            found_vulnerabilities = None
+
+                        if found_vulnerabilities is not None:
+                            keys = list(found_vulnerabilities)
+                            vuln = found_vulnerabilities[keys[FIRST_INDEX]]
```

Comment:
Any reason to only add the first one?

Reply:
I was thinking it might spam notifications if the index report contains a lot of vulnerabilities, but looking back I think we might as well add them all since the user can turn this on and off now.